.stylelintrc.yaml

```diff
@@ -0,0 +1,21 @@
+extends:
+  - stylelint-config-standard
+  - stylelint-config-prettier
+rules:
+  # Disallow color names and hex colors as these don't work well with dark mode.
+  # Use PF global variables instead:
+  # https://patternfly-react-main.surge.sh/developer-resources/global-css-variables#global-css-variables
+  color-named: never
+  color-no-hex: true
+  function-disallowed-list:
+    - rgb
+  # Disallow CSS classnames prefixed with .pf- or .co- as these prefixes are
+  # reserved by PatternFly and OpenShift console.
+  selector-disallowed-list: /\.(pf|co)-/
```

Comment:
This will disallow nested as well as partial SCSS joins?

eg
```
.pf {
  &-button { ... }
}
.custom.pf-button { ... }
```

Reply:
@spadgett Do you know when the linter runs? Is it on SCSS files or on the generated output as css? Because maybe neither of these cases are a problem.

The regexp looks to handle the 2nd case (not sure what I was thinking before) -- and the first case should be handled if it's run against the CSS output... but we could even just eliminate `.pf` and `.co` classes and it should cover the first case too.